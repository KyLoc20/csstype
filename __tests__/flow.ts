import { spawnSync } from 'child_process';
import * as path from 'path';

test('it detects errors', () => {
  const flowBin = path.resolve(
    __dirname,
    '../node_modules',
    '.bin',
    process.platform === 'win32' ? 'flow.cmd' : 'flow',
  );

  const fixturesDir = path.resolve(__dirname, '../__fixtures__');

  const args = ['check', fixturesDir];

  const result = spawnSync(flowBin, args, {
    stdio: 'pipe',
    encoding: 'utf8',
  });

  expect(result.stdout).toMatchSnapshot();
});
