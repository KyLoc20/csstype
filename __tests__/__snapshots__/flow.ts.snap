// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it detects errors 1`] = `"31:44 - Cannot assign object literal to \`unknownProperty\` because property \`unknownProperty\` is missing in \`Properties\` [1] but exists in object literal [2]."`;
